types/foundry-vtt-api/types/core/dicepool.d.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * A dice pool represents a set of Roll expressions which are collectively modified to compute an effective total
+ * across all Rolls in the pool. The final total for the pool is defined as the sum over kept rolls, relative to any
+ * success count or margin.
+ *
+ * @example
+ * // Consider 3 rolls
+ * let r1 = new Roll("4d6");
+ * let r2 = new Roll("3d8");
+ * let r3 = new Roll("2d10");
+ * // Keep the highest of the 3 roll expressions
+ * let pool = new DicePool([r1,r2,r3], "kh");
+ *
+ * @example
+ * // Construct a DicePool from a string formula
+ * let pool = DicePool.fromFormula("{4d6,3d8,2d10}kh");
+ */
+declare class DicePool {
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/core/dicepool.d.ts;
  Line: 19;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.